types/block.go

```diff
@@ -1,5 +1,11 @@
 package types
 
+import (
+	"errors"
+	abci "github.com/lazyledger/lazyledger-core/abci/types"
+	pb "github.com/lazyledger/optimint/types/pb/optimint"
```

Comment:
Sort imports

Reply:
My `:GoImports` used to work better... thanks for catching this!

Reply:
Actually, they are sorted, but I should add some empty lines